app/uk/gov/hmrc/lisaapi/connectors/DesConnector.scala

```diff
@@ -286,7 +285,9 @@ class DesConnector @Inject()(
         case SERVICE_UNAVAILABLE => DesUnavailableResponse
         case _ => parseDesResponse[DesTransactionResponse](res)
       }
-    })
+    }).recover{
```

Comment:
I'm not sure we want to turn every upstream error response into a DES unavailable? Ideally the recover block should replace the behaviour of the pattern matching on the status.